### PR TITLE
feat: add @chamaeleonidae/chmln types

### PR DESCRIPTION
**Pull request by borajimin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/chamaeleonidae__chmln/index.d.ts
1. types/chamaeleonidae__chmln/chamaeleonidae__chmln-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a2968cbd731c4a25ce112ebcda9b228bc7db8d25
</p>

File: types/chamaeleonidae__chmln/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        role?: string; // Send properties useful for targeting types of users (e.g. "Admin")
+        logins?: string; // Send any data about user engagement (e.g. 39)
+        project?: string; // Send any unique data for a user that might appear in any page URLs (e.g. 09876 or "12a34b56")
+
+        // OPTIONAL - Company properties:
+        company?: ChameleonCompanyOptions;
+    } & {
+        // OPTIONAL - Other extra properties:
+        [key: string]: string | undefined;
+    };
```
[the remark this answers is not in the document]
Comment (borajimin) [reply types/chamaeleonidae__chmln/index.d.ts:35]: When I tried sending a random field in the CompanyOptions, the property wasn't available on the user info view. So, I'm assuming the rules are more strict for CompanyOptions.

**Review by jakebailey — CHANGES_REQUESTED** [review]
After #67332, this PR will need to be updated to include an `.npmignore`. See https://github.com/DefinitelyTyped/DefinitelyTyped#npmignore

File: types/chamaeleonidae__chmln/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @definitelytyped/no-single-declare-module */
```
Comment (jakebailey) [inline types/chamaeleonidae__chmln/index.d.ts:1]: Why have you disabled these rules? This could all be at the top level.

**Comment by borajimin** [comment]
Closing since the chameleon team added their own.

**Pull request closed without merging**